.github/workflows/ci.yml

```diff
@@ -754,7 +754,7 @@ jobs:
           cargo build --manifest-path examples/maturin-starter/Cargo.toml --features generate-import-lib --target x86_64-pc-windows-gnu
           cargo xwin build --manifest-path examples/maturin-starter/Cargo.toml --features generate-import-lib --target x86_64-pc-windows-msvc
```

Comment:
Idea: try `--cross-compiler clang` to cross compile using [windows-msvc-sysroot](https://github.com/trcrsired/windows-msvc-sysroot) instead to avoid downloading MSVC SDKs from microsoft, not sure if it will work though.

Reply:
Perfect, works great 🎉